src/modules/headset/headset.h

```diff
@@ -28,7 +29,11 @@ typedef enum {
   DEVICE_HAND_RIGHT,
   DEVICE_EYE_LEFT,
   DEVICE_EYE_RIGHT,
-  MAX_DEVICES
+  DEVICE_GAMEPAD_1,
+  MAX_DEVICES,
+
+  DEVICE_GAMEPAD_FIRST = DEVICE_GAMEPAD_1,
+  DEVICE_GAMEPAD_LAST = DEVICE_GAMEPAD_1,
```

Comment:
The current gamepad driver accepts 1 gamepad at a time. If you have 2 plugged in and unplug 1, it promotes the other plugged-in one.

But notice, the way gamepad.c is written it can handle an arbitrary number of gamepad devices. The GLFW maximum number of gamepads is, I think, 14. I think having 4 gamepad devices would make a lot of sense. In my testing I had a problem because "spacemouse emulator" was showing up as a joystick device on Windows according to GLFW, so a real gamepad never got a chance to grab the GAMEPAD_1 slot (I uninstalled spacemouse for testing to work around this). If we supported 4 gamepads this would not have been a problem.